README.md

```diff
@@ -75,55 +64,32 @@ Currently not available
  - Have a code editor
 
 ### Setup:
-Clone this repository to your desired folder:
-
-
-- Clone this repository to your desired folder.
-- Run: git clone + 'link copied' and make sure the file is copied.
-- Open the file with a code edi
+```
+git clone https://github.com/johnie261/Book-store.git
+cd Book-store
+```
 
 ### Install:
- Install this project with:
-- Git.
-- Github.
+```
+npm install
+```
 
 ### Usage:
- To run the project, execute the following command:
-To run the project, execute the following command:
-
-
```

Comment:
![image](https://user-images.githubusercontent.com/93039359/229764397-202661ad-5045-4f80-ad84-6f2ede3f1b40.png)

**[OPTIONAL]**

- [ ] It would be very great if you make all your commit messages more meaningful and capitalize because I saw that some are not capitalize 😨 Kindly consider them making capitalize and meaningful to make your pull request professional 👍